pkg/apis/jenkins.io/v1/types_fact.go

```diff
@@ -0,0 +1,99 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:noStatus
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:openapi-gen=true
+
+// Fact represents observed facts
+type Fact struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object's metadata.
```

Comment:
Should there be resource links to the owning `PipelineActivity` for GC purposes?

Reply:
I added some docs on how to link the Facts to a PipelineActivity using labels.